src/west/app/project.py

```diff
@@ -232,7 +234,7 @@ def local(self, args) -> Path:
 
     def bootstrap(self, args) -> Path:
         manifest_url = args.manifest_url or MANIFEST_URL_DEFAULT
-        manifest_rev = args.manifest_rev or MANIFEST_REV_DEFAULT
+        manifest_rev = args.manifest_rev or self.get_head_branch(manifest_url)
```

Comment:
To be clear using `_head_` _in the source code_ looks good to me. Implementation details in the code are a good thing.